content/zh/_index.md

```diff
@@ -0,0 +1,24 @@
+
+---
+title: "云原生词汇表"
+---
+
+# 云原生词汇表
+
+云原生词汇表是由 CNCF 商业价值小组委员会 (BVS, Business Value Subcommittee) 领导的一个项目. 该项目目的是在不需要先验技术知识的情况下, 以清晰通俗的语言解释云原生概念. [您可以在这里查看或下载PDF版本](https://github.com/cncf/glossary/blob/main/cloudnative-glossary.pdf).
+
+## 社区贡献
+
+对本机词汇表的更改、添加和改进对所有人开放. 我们使用由 CNCF 管理的社区驱动流程来开发和完善这个共享词典. 该词汇表提供了一个厂商中立的平台, 用于管理云原生技术的共享词汇表. 欢迎所有参与者根据本项目的目的和章程作出贡献.
+
+任何想要贡献的人都可以提交 GitHub issue 或 创建 PR. 欲了解更多信息, 请查看[如何贡献](/zh/contribute/), 并且必须遵循[样式指南](/zh/style-guide/).
+
+## 致谢
+
+云原生词汇表可从 CNCF 营销委员会 (CNCF Marketing Committee) (商业价值小组委员会) 获得,
+包括 [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/),
+[Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/), [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), [Seokho Son](https://www.linkedin.com/in/seokho-son/) 以及更多的贡献者. 如想获得完整的贡献者名单, 请跳转至 [GitHub Pages 页](https://github.com/cncf/glossary/graphs/contributors).
+
```

Comment:
Hello, @Jacob953 
Please add acknowledgement to localization contributors to _index.md. 
(similar style with ack to Glossary contributors)

You can open another PR to add it.

Reply:
Thx for the reminder! @seokho-son 
I've handled this in #869 ;)